lstchain/calib/camera/r0.py

```diff
@@ -418,8 +421,8 @@ def __init__(self, **kwargs):
                       "the R0 level, then r1 samples will equal r0 samples")
 
     def calibrate(self, event):
-        for telid in event.r0.tels_with_data:
-            event.r1.tel[telid].trigger_type = event.r0.tel[telid].trigger_type
-            event.r1.tel[telid].trigger_time = event.r0.tel[telid].trigger_time
-            samples = event.r0.tel[telid].waveform[:,:,self.r1_sample_start:self.r1_sample_end]
-            event.r1.tel[telid].waveform = samples.astype('float32')- self.offset
+        for tel_id in event.r0.tels_with_data:
+            event.r1.tel[tel_id].trigger_type = event.r0.tel[tel_id].trigger_type
+            event.r1.tel[tel_id].trigger_time = event.r0.tel[tel_id].trigger_time
+            samples = event.r0.tel[tel_id].waveform[:, :, self.r1_sample_start:self.r1_sample_end]
+            event.r1.tel[tel_id].waveform = samples.astype('float32') - self.offset
```

Comment:
Why type suddenly becomes `float32` while it was `int16` before ?
R1 is stored as `int16` right?

Reply:
After call

```
lst_r0.subtract_pedestal(ev)
lst_r0.time_lapse_corr(ev)
lst_r0.interpolate_spikes(ev)
``` 
R1 is store as `int16`, but after call `lst_r0.calibrate(ev)` as `float32`.
This is mistake in my code. I am sorry.
Thanks @vuillaut for report.
Dear @garciagenrique could you change `samples.astype('float32')` to `samples.astype('int16')` or I can do it in next PR.

Reply:
Sure @pawel21, I can change it now.